<log4j:event logger="ASP.global_asax" timestamp="1415129856609" level="ERROR" thread="16"><log4j:message>Unhandled Exception with request (ReturnUrl=%2fKayflow%2fdefault.aspx);</log4j:message><log4j:properties><log4j:data name="log4jmachinename" value="SweetHome" /><log4j:data name="log4japp" value="/LM/W3SVC/1/ROOT/Kayflow-1-130596034373345424" /><log4j:data name="log4net:UserName" value="NT AUTHORITY\—»—“≈ћј" /><log4j:data name="log4net:Identity" value="" /><log4j:data name="log4net:HostName" value="SweetHome" /></log4j:properties><log4j:throwable>System.Data.SqlClient.SqlException (0x80131904): ѕри установлении соединени€ с SQL Server произошла ошибка, св€занна€ с сетью или с определенным экземпл€ром. —ервер не найден или недоступен. ”бедитесь, что им€ экземпл€ра указано правильно и что на SQL Server разрешены удаленные соединени€. (provider: SQL Network Interfaces, error: 26 - ќшибка при обнаружении указанного сервера или экземпл€ра)
   в System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   в System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   в System.Data.SqlClient.TdsParser.Connect(ServerInfo serverInfo, SqlInternalConnectionTds connHandler, Boolean ignoreSniOpenTimeout, Int64 timerExpire, Boolean encrypt, Boolean trustServerCert, Boolean integratedSecurity, Boolean withFailover)
   в System.Data.SqlClient.SqlInternalConnectionTds.AttemptOneLogin(ServerInfo serverInfo, String newPassword, SecureString newSecurePassword, Boolean ignoreSniOpenTimeout, TimeoutTimer timeout, Boolean withFailover)
   в System.Data.SqlClient.SqlInternalConnectionTds.LoginNoFailover(ServerInfo serverInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString connectionOptions, SqlCredential credential, TimeoutTimer timeout)
   в System.Data.SqlClient.SqlInternalConnectionTds.OpenLoginEnlist(TimeoutTimer timeout, SqlConnectionString connectionOptions, SqlCredential credential, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance)
   в System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData)
   в System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal&amp; connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в Framework.SqlDataAccess.Controller.StoredProcedure.OpenConnection() в d:\Projects\SqlDataAccess\Framework.SqlDataAccess.Controller\DALC\StoredProcedure.cs:строка 57
   в Framework.SqlDataAccess.Controller.StoredProcedure.ExecuteReader() в d:\Projects\SqlDataAccess\Framework.SqlDataAccess.Controller\DALC\StoredProcedure.cs:строка 179
   в Framework.SqlDataAccess.Controller.SQLDataAccessExtensions.ExecuteListAttr[T](StoredProcedure st) в d:\Projects\SqlDataAccess\Framework.SqlDataAccess.Controller\DALC\Extensions.cs:строка 203
   в Framework.SqlDataAccess.Controller.ReadControllerSinglePK`2.GetAll(String orderFieldName, Boolean hideDeleted, Object notHiddenID) в d:\Projects\SqlDataAccess\Framework.SqlDataAccess.Controller\ControllerBase.cs:строка 306
   в Framework.SqlDataAccess.Controller.ReadControllerSinglePK`2.GetAll(String orderFieldName, Boolean hideDeleted) в d:\Projects\SqlDataAccess\Framework.SqlDataAccess.Controller\ControllerBase.cs:строка 294
   в Kayflow.Manager.KayflowManager`2.GetAll(String orderFieldName, Boolean hideDeleted) в d:\Projects\Kailas\Kayflow\Source\Kayflow.Manager\Common.cs:строка 96
   в Kayflow.Manager.KayflowManager`2.GetAll() в d:\Projects\Kailas\Kayflow\Source\Kayflow.Manager\Common.cs:строка 88
   в Kayflow.Manager.EmployeeManager.CheckLogin(String pLogin, String pPassword, Guid pOfficeID) в d:\Projects\Kailas\Kayflow\Source\Kayflow.Manager\Staff\EmployeeManager.cs:строка 50
   в Controls_LoginControl.clbkSave_OnCallback(Object source, CallbackEventArgs e) в d:\Projects\Kailas\Kayflow\Source\Kayflow.Web\Controls\LoginControl.ascx.cs:строка 23
   в DevExpress.Web.ASPxCallback.ASPxCallback.RaiseCallbackEvent(String eventArgument)
   в DevExpress.Web.ASPxClasses.ASPxWebControl.System.Web.UI.ICallbackEventHandler.RaiseCallbackEvent(String eventArgument)
ClientConnectionId:00000000-0000-0000-0000-000000000000</log4j:throwable><log4j:locationInfo class="ASP.global_asax" method="Application_Error" file="d:\Projects\Kailas\Kayflow\Source\Kayflow.Web\Global.asax" line="45" /></log4j:event>
